granian/server.py

```diff
@@ -62,6 +62,13 @@ def terminate(self):
         self.interrupt_by_parent = True
         self.proc.terminate()
 
+    def kill(self):
+        self.interrupt_by_parent = True
+        self.proc.kill()
+
+    def is_alive(self):
+        return self.proc.is_alive()
```

Comment:
I see this defined, but then in later code you call `this.proc.is_alive()`. So either:
- drop this
- use it in code (and at that point can probably be a `property`?)